crates/epaint/src/text/fonts.rs

```diff
@@ -254,6 +254,34 @@ pub struct FontDefinitions {
     pub families: BTreeMap<FontFamily, Vec<String>>,
 }
 
+#[derive(Debug, Clone)]
+pub struct FontInsert {
+    /// Font name
+    pub name: String,
+
+    /// A `.ttf` or `.otf` file and a font face index.
+    pub data: FontData,
+
+    /// Sets the font family
+    pub family: FontFamily,
```

Comment:
You sometimes want to install the font for multiple families. For instance, when adding support for Chinese, you may want to set it as the primary font for `Proportional` and as fallback for `Monospace`